Amazon.QLDB.Driver/transaction/Transaction.cs

```diff
@@ -106,10 +109,13 @@ internal virtual IResult Execute(string statement, List<IIonValue> parameters)
                 parameters = new List<IIonValue>();
             }
```

Comment:
Can we replace the pattern of 
if (x == null) {
 x = not null
}
with
`x ??= not null`

Reply:
Done